### PR TITLE
Fixes #147, fix probabilities when multi_class == 'multinomial'

### LINKED ISSUE #147: Logistic Regression probability scores wrong in case of string labels and multi_class set to multinomial.
data = load_digits()
X = data.data
y = data.target

X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.5, random_state=42)
model = LogisticRegression(random_state=42, multi_class='multinomial', solver='sag').fit(X_train, y_train)

onnx_model = convert_sklearn(model, 'lr', [('input', FloatTensorType(X_test.shape))])
save_model(onnx_model, 'lr.onnx')
sess = InferenceSession('lr.onnx')
res = sess.run(None, input_feed={'input': X_test.astype(np.float32)})

np.mean(np.isclose(model.predict_proba(X_test), list(map(lambda x:
                                                         list(map(lambda y: x[y], x)), res[1]))))

>> 0.010901001112347052

### PR DESCRIPTION
**Pull request by xadupre**

### CODE REVIEW

File: skl2onnx/operator_converters/linear_classifier.py [hunk 1]
```diff
@@ -53,11 +59,11 @@ def convert_sklearn_linear_classifier(scope, operator, container):
...
                 """ # noqa
-                classifier_attrs['post_transform'] = 'LOGISTIC'
+                classifier_attrs['post_transform'] = post_transform
             else:
-                classifier_attrs['post_transform'] = 'LOGISTIC'
+                classifier_attrs['post_transform'] = post_transform
         else:
-            classifier_attrs['post_transform'] = 'LOGISTIC'
+            classifier_attrs['post_transform'] = post_transform
     elif op.__class__.__name__ in ('LinearSVC'):
```
Comment (prabhat00155) [inline skl2onnx/operator_converters/linear_classifier.py]: This is not needed as we already checked for LinearSVC above.

File: skl2onnx/operator_converters/linear_classifier.py [hunk 2]
```diff
@@ -53,11 +59,11 @@ def convert_sklearn_linear_classifier(scope, operator, container):
                 is not SOFTMAX.
                 https://github.com/scikit-learn/scikit-learn/blob/bac89c253b35a8f1a3827389fbee0f5bebcbc985/sklearn/linear_model/base.py#L300
                 """ # noqa
-                classifier_attrs['post_transform'] = 'LOGISTIC'
+                classifier_attrs['post_transform'] = post_transform
```
Comment (prabhat00155) [inline skl2onnx/operator_converters/linear_classifier.py]: Why do we have three same statements with if-else? Can't we just remove all the if-else and just have the assignment statement?

File: skl2onnx/operator_converters/linear_classifier.py (hunk 1 above)
Comment (prabhat00155) [reply skl2onnx/operator_converters/linear_classifier.py]: Can you fix this?

File: skl2onnx/operator_converters/linear_classifier.py (hunk 2 above)
Comment (xadupre) [reply skl2onnx/operator_converters/linear_classifier.py]: Fixed.

File: skl2onnx/operator_converters/linear_classifier.py (hunk 1 above)
Comment (xadupre) [reply skl2onnx/operator_converters/linear_classifier.py]: I think I did.

Comment (prabhat00155) [reply skl2onnx/operator_converters/linear_classifier.py]: I don't see any difference between lines 47-48 and 55-56 in this file.

Comment (xadupre) [reply skl2onnx/operator_converters/linear_classifier.py]: My mistake. Fixed.
